content/posts/2014-09-04-a-new-look-at-the-freedom-of-information-act.md

```diff
@@ -5,8 +5,6 @@ tumblr_url: http://18fblog.tumblr.com/post/96625804492/a-new-look-at-the-freedom
 
 title: "A new look at the Freedom of Information Act"
 
-image: /assets/blog/foia/in-hand.jpg
```

Comment:
Image data from the frontmatter can be used in other places, so let's keep any `image: ` frontmatter — unless I've missed something and there's a reason not to.

Reply:
Restored in https://github.com/18F/18f.gsa.gov/pull/3863/commits/ef47484d22fc375a653dac756676b07b69149b86#diff-4eb32b78667fa986504034f17af55b60909e4738dca56c54bcda5cd7a930c42bR5!